doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +260,95 @@ To build and program the source code from the command line, complete the followi
          west flash
 
    The device resets and runs the programmed sample or application.
+
+.. _thingy53_app_update:
+
+Updating firmware image
+=======================
+
+The firmware on Thingy:53 can be programmed using an external debug probe and 10-pin JTAG cable, as described in sections related to building in various environments.
```

Comment:
Maybe just link to a section instead of mentioning it like that:
"as described in sections related to building in various environments" ?

Reply:
This is referring to the Building and programmin with SEGGER", "...with Visual Studio Code", and "...on the command line" just above this section. Something like this could work:
```suggestion
The firmware on Thingy:53 can be programmed using an external debug probe and 10-pin JTAG cable, as described in the sections above for building in different environments.
```